### PR TITLE
voice avatar setup

### PR DESCRIPTION
**Pull request by mritunjayz**

### CODE REVIEW

File: web/src/components/pages/profile/avatar-setup/avatar-setup.css [hunk 1]
```diff
@@ -1,8 +1,64 @@
...
+    flex-direction: row;
+    flex-wrap: wrap;
+    max-width: var(--wide-desktop-width);
+}
+
+.clip {
+    & h2 {
+        width: 334px;
+        height: 58px;
+        font-family: ZillaSlab;
```
Comment (Gregoor) [inline web/src/components/pages/profile/avatar-setup/avatar-setup.css]: can we use our css variables for fonts here `--base-font-family` and `--strong-font-family` if I remember correctly

Comment (mritunjayz) [reply web/src/components/pages/profile/avatar-setup/avatar-setup.css]: :+1:

File: web/src/components/pages/profile/avatar-setup/avatar-setup.tsx [hunk 2]
```diff
@@ -19,17 +19,16 @@ import {
...
+import { Voice } from '../../../primary-buttons/primary-buttons';
 import AudioIOS from '../../contribution/speak/audio-ios';
 import AudioWeb, { AudioError } from '../../contribution/speak/audio-web';
 import { isFirefoxFocus, isNativeIOS, isProduction } from '../../../../utility';
+import { Suspense, lazy } from 'react';
+const Lottie = lazy(() => import('react-lottie'));
+const animationData = require('../../../layout/data.json');
 
 import './avatar-setup.css';
+import { string, object } from 'prop-types';
```
Comment (Gregoor) [inline web/src/components/pages/profile/avatar-setup/avatar-setup.tsx]: Why do we import PropTypes here? Don't we do all the typing with TS here?

Comment (mritunjayz) [reply web/src/components/pages/profile/avatar-setup/avatar-setup.tsx]: Ah, maybe this is added mistakenly by me while editor was showing some options of auto-fix any problem

Comment (Gregoor) [reply web/src/components/pages/profile/avatar-setup/avatar-setup.tsx]: Makes sense! Should be safe to remove

**Comment by Gregoor** [comment]
LGTM! Should I merge this in?

**Comment by mritunjayz** [comment]
:+1:

**Pull request merged**